src/styles/main.scss

```
@@ -1,3 +1,91 @@
+$gap-size: 1px;
+$calendar-margin: auto;
+$font-size: 30px;
+$border-size: 1px;
+$wigth-cell: 100px;
+$height-cell: 100px;
```

you don't need 2 variables with the same values
```suggestion
$cell-size 100px;
```